Robert Khromov/src/components/Blog.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Post from './Post'
+
+export default class Blog extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            posts: []
+        }
+    }
+
+
+    render() {
+        this.state.posts = this.props.posts.map((i) => {
```

Comment:
```suggestion
        this.state.posts = this.props.posts.map((post) => {
```
давайте содержательные названия:) в данном случае мы работаем с отдельным постом